rhobs/olm/manifests/admission-webhook/alertmanager-config-validating-webhook.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/component: admission-webhook
+    app.kubernetes.io/name: prometheus-operator-admission-webhook
+  name: alertmanagerconfigs.monitoring.rhobs
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    # NOTE: the caBundle get automatically injected by OLM
+    caBundle: Cg==
+    service:
+      # NOTE: when changing the service, ensure the same changes are applied
+      # to prometheus-rule-validating-webhook
+      name: rhobs-prometheus-operator-admission-webhook
```

Comment:
this service name generated from obo is `obo-prometheus-operator-admission-webhook`